src/class-post-republisher.php

```diff
@@ -335,4 +337,22 @@ public function add_rewrite_schedule_display_state( $post_states, $post ) {
 
 		return $post_states;
 	}
+
+	/**
+	 * Deletes the original post meta that flags it has having a copy when the copy is manually deleted.
```

Comment:
Small type: `has having` should be `as having`.

Reply:
Will fix.